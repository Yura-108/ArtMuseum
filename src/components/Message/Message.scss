<<<<<<< HEAD
@import '@styles/_mixins.scss';
@import '@styles/_variables.scss';

.message {
  color: $color-text-grey;
  @include font-style('Lexend Deca', 400, 32px, 40px);
  @media (max-width: $width-main-container) {
    @include font-style('Lexend Deca', 400, 24px, 24px);
  }

  @media (max-width: $width-main-container-small) {
    @include font-style('Lexend Deca', 400, 16px, 16px);
=======
.message {
  color: rgb(57, 57, 57);
  font-family: 'Lexend Deca', sans-serif;
  font-weight: 400;
  font-size: 32px;
  line-height: 40px;
  @media (max-width: 1024px) {
    font-size: 24px;
    line-height: 24px;
  }

  @media (max-width: 768px) {
    font-size: 16px;
    line-height: 16px;
>>>>>>> main
  }
}
