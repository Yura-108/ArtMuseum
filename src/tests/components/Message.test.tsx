import { render, screen } from '@testing-library/react';
import Message from '@components/Message/Message.tsx';

describe('Message', () => {
  it('renders the Message component with children', () => {
    render(<Message>Test Message</Message>);

    const messageElement = screen.getByRole('heading', { level: 2 });
    expect(messageElement).toBeInTheDocument();
    expect(messageElement).toHaveTextContent('Test Message');
    expect(messageElement).toHaveClass('message');
  });

  it('renders correctly with different text', () => {
    render(<Message>Another Message</Message>);

    expect(screen.getByRole('heading', { level: 2 })).toHaveTextContent('Another Message');
  });

  it('matches snapshot', () => {
    const { asFragment } = render(<Message>Snapshot Message</Message>);

    expect(asFragment()).toMatchSnapshot();
  });
});
