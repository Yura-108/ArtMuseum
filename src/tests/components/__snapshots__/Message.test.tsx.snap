// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message matches snapshot 1`] = `
<DocumentFragment>
  <h2
    class="message"
  >
    Snapshot Message
  </h2>
</DocumentFragment>
`;
